tests/console/yast2_samba.pm

```diff
@@ -0,0 +1,279 @@
+    send_key 'alt-a';
+    sleep 1;
+    for (1 .. 20) { send_key 'backspace'; }
+    type_string 'cn=admin';
+    send_key 'alt-l';
+    type_string 'testing';
+    sleep 1;
+    send_key 'alt-v';
+    type_string 'testing';
+    sleep 1;
+    # use database as default for ldap client
+    send_key 'alt-u';
+    sleep 1;
+    send_key 'alt-n';
+    sleep 1;
+    assert_screen 'yast2_ldap_configuration_kerberos';
+    send_key 'alt-x';
+    sleep 1;
+    assert_screen 'yast2_ldap_configuration_summary';
+
+    # finish ldap server configuration
+    send_key 'alt-f';
+    sleep 10;
+
+    # check ldap server status at first, a local ldap server is needed in the test case
+    assert_script_run "systemctl show -p ActiveState slapd.service | grep ActiveState=active";
+
+    # install samba stuffs at first
+    assert_script_run("/usr/bin/zypper -n -q in samba samba-client yast2-samba-server");
+
+    # start samba server configuration
+    script_run("/sbin/yast2 samba-server; echo yast2-samba-server-status-\$? > /dev/$serialdev", 0);
+
+    # check Samba-Server Configuration got started
+    assert_screen 'yast2_samba_installation';
+    send_key 'alt-w';
+    for (1 .. 12) { send_key 'backspace'; }
+    # give a new name for Workgroup
+    type_string 'QA-Workgroup';
+    assert_screen 'yast2_samba-server_workgroup_new';
+    send_key 'alt-n';
+
+    # select "Not a Domain Controller"
+    assert_screen 'yast2_samba_server_selection';
+    send_key 'alt-p';
+    sleep 1;
+    send_key 'alt-a';
+    sleep 1;
+    send_key 'alt-c';
+
+    # check "Not a DC" is select
+    assert_screen 'yast2_samba-server_not-a-dc_selected';
+    send_key 'alt-n';
+    sleep 5;
+    # now move to Samba configuration and check enable service start during boot
+    send_key 'alt-r';
+    assert_screen 'yast2_samber-server_start-during-boot';
+
+    # open port in firewall if it is eanbaled and check network interfaces, check long text by send key right.
+    if (assert_screen 'yast2_samba_open_port_firewall') {
+        send_key 'alt-f';
+        sleep 1;
+        send_key 'alt-i';
+        assert_screen 'yast2_samba_firewall_port_details';
+        send_key 'alt-e';
+        for (1 .. 5) { send_key 'right'; }
+        send_key 'alt-a';
+        send_key 'alt-o';
+    }
+
+    # swith to Samba Configuration - Shares
+    send_key 'alt-s';
+    assert_screen 'yast2_samba-server_shares';
+
+    # add a shares config html_public
+    send_key 'alt-a';
+    assert_screen 'yast2_samba-server_new-share';
+    send_key 'alt-n';
+    type_string 'html_public';
+    sleep 1;
+    send_key 'alt-a';
+    type_string 'html docs for share';
+
+    # select share type as directory and give a new share path /home/html_public
+    send_key 'alt-d';
+    sleep 1;
+    send_key 'alt-s';
+    for (1 .. 8) { send_key 'backspace'; }
+    type_string '/home/html_public';
+
+    # set read-only and utilize brtfs features
+    send_key 'alt-r';
+    sleep 1;
+    # send_key 'alt-u';
```

Comment:
The code is disabled by the use of a comment therefore it has no effect. 
Nobody else will know when this should be enabled so better just delete the 
line if your test works like this.